scanner/templates/go/Dockerfile

```
@@ -0,0 +1,15 @@
+ARG GO_VERSION=1
+FROM golang:${GO_VERSION} as builder
```

if the runner is alpine, the builder has to be alpine based too
```suggestion
FROM golang:${GO_VERSION}-alpine as builder
```